### DISCUSSION TITLE
744

### DISCUSSION THREAD

**Discussion by giscus** [jojoldu/blog-comments#925] [Comments]
# 744

퇴근 후에 깨끗하게 샤워하고 거실 불을 끄고, 테이블 전등만 켠 채로 적당히 늘어지는 음악도 들으면서 코드 작업이나 글 쓰는 시간을 좋아한다. 오늘은 유달리 날이 되게 추운데 바깥 야경이 또렷하게 보였다. 반짝반짝 빛나는 야경은 왜 이렇게 예쁜걸까. 언젠가 이 집을 떠나는 날이 오더라도, 매일 근사한 감동을 주었던 이 밤들을 잊지 못할것 같다. 오늘은 무슨 노래를 들을까 하면서 유튜브를 찾다가 우연히 배우 알렝 드롱의 리즈시절에 관한 쇼츠를 보게 되었다. 여기서 나온 노래가 너무 마음에 드는 것이다. 적당히 늘어지고, 뭔가 7080 같은 느낌도 들고 말이다. 그래서 좀 더 찾아보니 Mild High Club 의 Homeage 란 노래였다. 노래가 너무 몽환적이라서 무슨 가사일까 궁금했다. Someone ..

https://jojoldu.tistory.com/744

**Comment by Hoo93** [comment]
좋은 책들도 자주 소개해 주시면 좋겠어요. 유튜브나 블로그에 소개해 주시는 책들 보고 열심히 공부하고 감명도 많이 받았습니다
